### ISSUE TITLE
Hangs when using FakeReadout mode on Github

### ISSUE THREAD

**Issue by chinaar** [kkapsner/CanvasBlocker#42]
When using Nightly x64 2015-11-01 e10s on Windows 10 x64, I am getting hangs when accessing Github (and other pages which I assume are also making Canvas calls) using the "fake Readout API" setting. These hangs immediately disappear on my browser when I set the extension to the "block Readout API" setting.

**Comment by facetoe** [comment]
I also experienced this issue running:

```
Linux facetoe 4.2.4-1-ARCH #1 SMP PREEMPT Fri Oct 23 07:32:38 CEST 2015 x86_64 GNU/Linux
```

with:

```
Mozilla Firefox 44.0a1
```
```

**Comment by kkapsner** [comment]
I can reproduce the problem. It is created by e10s - seems like my fake option does not work well with the multi process infrastructure. Until I figure out the exact problem you can disable electrolysis: https://wiki.mozilla.org/Electrolysis#Enabling_and_Disabling_Electrolysis

**Comment by kkapsner** [comment]
This is fixed with version 0.3.0
Please test!
